server/planning/assignments/assignments.py

```diff
@@ -1299,17 +1299,12 @@ def _publish_planning(item):
             try:
                 # check if the planning item is locked
                 lock_service.validate_relationship_locks(planning_item, "planning")
-                use_published_planning = False
             except SuperdeskApiError as ex:
                 # planning item is already locked.
-                use_published_planning = True
-                logger.exception(str(ex))
+                lock_service = get_component(LockService)
+                lock_service.unlock(planning_item, user_id, get_auth()["_id"], "planning")
```

Comment:
I wonder if we need to unlock or deal with locks at all when updating assignment